lib/assets/stylesheets/modules/_collapsible.scss

```diff
@@ -14,7 +14,7 @@
 .collapsible__toggle {
   position: absolute;
   top: 0;
-  right: -25px;
+  right: $govuk-gutter-half * -1;
```

Comment:
The `gutter` variables from Frontend are only for the space in between the grid – ideally this would use the spacing scale, or continue to use a hardcoded value.

(unless this is dependant on the grid, in which case I'd add a comment explaining)

Reply:
This was the only other visual regression I found, this changes the value and moves the placement of the toggle button. I'd recommend using the previous value.